app/features/receive/receive-cashu.tsx

```diff
@@ -179,64 +112,31 @@ function MintQuoteItem({ account, amount, isVisible }: MintQuoteProps) {
   );
 }
 
-function useCarousel() {
-  const [api, setApi] = useState<CarouselApi>();
-  const [current, setCurrent] = useState(0);
-
-  useEffect(() => {
-    if (!api) return;
-    setCurrent(api.selectedScrollSnap());
-    api.on('select', () => setCurrent(api.selectedScrollSnap()));
-  }, [api]);
-
-  const scrollToIndex = (index: number) => {
-    api?.scrollTo(index);
-  };
-
-  return { current, scrollToIndex, setApi };
-}
-
-/** Two buttons to flip back and forth between the two QR codes. */
-function CarouselControls({
-  current,
-  onSelect,
-}: {
-  current: number;
-  onSelect: (index: number) => void;
-}) {
-  return (
-    <div className="mt-8 flex flex-col items-center gap-4">
-      <div className="flex rounded-full border">
-        <button
-          type="button"
-          className={`rounded-full px-6 py-3 ${
-            current === 0 ? 'bg-primary text-primary-foreground' : ''
-          }`}
-          onClick={() => onSelect(0)}
-        >
-          <Banknote className="h-5 w-5" />
-        </button>
-        <button
-          type="button"
-          className={`rounded-full px-6 py-3 ${
-            current === 1 ? 'bg-primary text-primary-foreground' : ''
-          }`}
-          onClick={() => onSelect(1)}
-        >
-          <Zap className="h-5 w-5" />
-        </button>
-      </div>
-    </div>
-  );
-}
+const carouselControls: Control[] = [
```

Comment:
this seems like an overkill if we are only having two controls and it's not likely there will be more. I'd just define this directly in jsx